public/components/JoinDataTable.vue

```diff
@@ -1,51 +1,48 @@
 <template>
-  <fixed-header-table ref="fixedHeaderTable">
-    <b-table
-      bordered
-      hover
-      small
-      :items="items"
-      :fields="emphasizedFields"
-      @sort-changed="onSortChanged"
-      @head-clicked="onColumnClicked"
+  <b-table
+    bordered
+    hover
+    small
+    :items="items"
+    :fields="emphasizedFields"
+    @sort-changed="onSortChanged"
```

Comment:
Here you kept the function to sort, but the method is gone.

Reply:
Ah , wait I see what you mean, I need to cut that listener.